<launch>
	<arg
		name="device_name"
		default="mettler_xpr226"/>
	<arg
		name="connection_mode"
		default="serial"/>
	<arg
		name="address"
		default="None"/>
	<arg
		name="port"
		default="/dev/ttyUSB0"/>
	<node
		name="mettler_xpr226_drq"
		pkg="roslabware_drivers"
		type="mettler_xpr226_drq"
		args="
			-d $(arg device_name)
			-c $(arg connection_mode)
			-add $(arg address)
			-p $(arg port)
			-l $(arg simulation)"
		output="screen"/>
</launch>
